File: src/main/java/com/fatec/destino/dto/compra/ViagemResumoDTO.java
package com.fatec.destino.dto.compra;

import com.fasterxml.jackson.annotation.JsonFormat;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.util.UUID;

public record ViagemResumoDTO(
        UUID id,
        Long pacoteId,
        String nomePacote,
        String descricao,
        BigDecimal valor,
        String statusCompra,
        @JsonFormat(pattern = "dd-MM-yyyy")
        LocalDate dataPartida,
        @JsonFormat(pattern = "dd-MM-yyyy")
        LocalDate dataRetorno,
        String imagemCapa,
        String cidade,
        String estado
) {}

File: src/main/kotlin/com/fatec/destino/dto/compra/ViagemResumoDTO.kt
package com.fatec.destino.dto.compra

import com.fatec.destino.model.pacote.hotel.cidade.Cidade
import com.fatec.destino.util.model.compra.StatusCompra
import java.math.BigDecimal
import java.time.LocalDate
import java.util.UUID

data class ViagemResumoDTO(
    val id: UUID?,
    val pacoteId: Long?,
    val nomePacote: String?,
    val descricao: String?,
    val valor: BigDecimal?,
    val statusCompra: StatusCompra?,
    val dataPartida: LocalDate?,
    val dataRetorno: LocalDate?,
    val imagemCapa: String?,
    val cidade: Cidade?,
    val estado: String?
)
